docs/resources/managed_database_mysql.md

```diff
@@ -0,0 +1,155 @@
+- **binlog_retention_period** (Number) The minimum amount of time in seconds to keep binlog entries before deletion. This may be extended for services that require binlog entries for longer than the default for example if using the MySQL Debezium Kafka connector.
+- **connect_timeout** (Number) connect_timeout
+- **default_time_zone** (String) default_time_zone
+- **group_concat_max_len** (Number) group_concat_max_len
+- **information_schema_stats_expiry** (Number) information_schema_stats_expiry
+- **innodb_ft_min_token_size** (Number) innodb_ft_min_token_size
+- **innodb_ft_server_stopword_table** (String) innodb_ft_server_stopword_table
+- **innodb_lock_wait_timeout** (Number) innodb_lock_wait_timeout
+- **innodb_log_buffer_size** (Number) innodb_log_buffer_size
+- **innodb_online_alter_log_max_size** (Number) innodb_online_alter_log_max_size
+- **innodb_print_all_deadlocks** (Boolean) innodb_print_all_deadlocks
+- **innodb_rollback_on_timeout** (Boolean) innodb_rollback_on_timeout
+- **interactive_timeout** (Number) interactive_timeout
+- **internal_tmp_mem_storage_engine** (String) internal_tmp_mem_storage_engine
+- **ip_filter** (List of String) IP filter
+- **long_query_time** (Number) long_query_time
+- **max_allowed_packet** (Number) max_allowed_packet
+- **max_heap_table_size** (Number) max_heap_table_size
+- **migration** (Block List, Max: 1) Migrate data from existing server (see [below for nested schema](#nestedblock--properties--migration))
+- **net_read_timeout** (Number) net_read_timeout
+- **net_write_timeout** (Number) net_write_timeout
+- **public_access** (Boolean) Public Access
+- **slow_query_log** (Boolean) slow_query_log
+- **sort_buffer_size** (Number) sort_buffer_size
+- **sql_mode** (String) sql_mode
+- **sql_require_primary_key** (Boolean) sql_require_primary_key
+- **tmp_table_size** (Number) tmp_table_size
+- **version** (String) MySQL major version
+- **wait_timeout** (Number) wait_timeout
+
+<a id="nestedblock--properties--migration"></a>
+### Nested Schema for `properties.migration`
+
+Optional:
+
+- **dbname** (String) Database name for bootstrapping the initial connection
+- **host** (String) Hostname or IP address of the server where to migrate data from
+- **ignore_dbs** (String) Comma-separated list of databases, which should be ignored during migration (supported by MySQL only at the moment)
+- **password** (String, Sensitive) Password for authentication with the server where to migrate data from
+- **port** (Number) Port number of the server where to migrate data from
+- **ssl** (Boolean) The server where to migrate data from is secured with SSL
+- **username** (String) User name for authentication with the server where to migrate data from
+
+
+
+<a id="nestedatt--components"></a>
+### Nested Schema for `components`
+
+Read-Only:
+
+- **component** (String)
```

Comment:
Should we include some descriptions here? Or is it something that a mysql-versed person should know anyway?

Reply:
Good catch thanks! It seems that doc generator ignores `components` and also `node_states` field descriptions. I'll check if there is something wrong with schema or is this just doc gen. bug.

Reply:
This seems to be known bug in tfplugindocs [#28](https://github.com/hashicorp/terraform-plugin-docs/issues/28). Those are read-only informational fields. I think that  in the future we can handle documentation in separate PR if needed.